src/pages/cohorts.tsx

```diff
@@ -28,55 +33,62 @@ type UserDetails = {
   centers?: any;
   Programs?: any;
 };
+
+// colums in table
 const columns = [
-  // {
-  //   key: "userId",
-  //   title: "ID",
-  //   dataType: DataType.String,
-  // },
   {
     key: "cohortName",
     title: "Name",
     dataType: DataType.String,
     sortDirection: SortDirection.Ascend,
   },
-  // {
-  //   key: "centers",
-  //   title: "Centers",
-  //   dataType: DataType.String,
-  // },
-  // {
-  //   key: "programs",
-  //   title: "Programs",
-  //   dataType: DataType.String,
-  // },
   {
     key: "actions",
     title: "Actions",
     dataType: DataType.String,
   },
 ];
 const Cohorts: React.FC = () => {
+  // use hooks
+  const { t } = useTranslation();
+  // handle states
   const [selectedState, setSelectedState] = React.useState<string[]>([]);
   const [selectedDistrict, setSelectedDistrict] = React.useState<string[]>([]);
   const [selectedBlock, setSelectedBlock] = React.useState<string[]>([]);
   const [selectedSort, setSelectedSort] = useState("Sort");
   const [pageOffset, setPageOffset] = useState(0);
   const [pageLimit, setPageLimit] = useState(10);
   const [selectedFilter, setSelectedFilter] = useState("All");
-
-  const [data, setData] = useState<UserDetails[]>([]);
-  const { t } = useTranslation();
+  const [cohortData, setCohortData] = useState<UserDetails[]>([]);
   const [pageSize, setPageSize] = React.useState<string | number>("");
   const [open, setOpen] = React.useState(false);
   const [confirmationModalOpen, setConfirmationModalOpen] =
     React.useState<boolean>(false);
-
   const [selectedCohortId, setSelectedCohortId] = React.useState<string>("");
   const [editModelOpen, setIsEditModalOpen] = React.useState<boolean>(false);
   const [confirmButtonDisable, setConfirmButtonDisable] =
     React.useState<boolean>(false);
   const [inputName, setInputName] = React.useState<string>("");
+  const [cohortName, setCohortName] = React.useState<string>("");
+  const [loading, setLoading] = useState<boolean | undefined>(undefined);
+  const [userId, setUserId] = useState("");
+
+  // use api calls
+  useEffect(() => {
+    if (typeof window !== "undefined" && window.localStorage) {
+      const user_Id = localStorage.getItem("userId") || "";
```

Comment:
@upendraTekdi dont use underscore in variable name

Reply:
done